cookbooks/bcpc/recipes/graphite_handler.rb

```diff
@@ -5,20 +5,23 @@
 # This recipe should be the first recipe in the run list for all the
 # nodes to collect chef-client stats and store into Graphite
 #
+simple_graphite_version = '>=2.1'
```

Comment:
The version was already pinned there.  I just extracted it to a variable because I used it twice.

Many versions are pinned in the Chef code, and that was not the problem I am trying to fix.